.circleci/marvin-analyze.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
```

Comment:
I think maybe create a subdir `marvin` inside `.circleci` and put all marvin scripts inside it. And the package.json file.

Reply:
Ok, but only in the next PR (which is almost ready), because doing it here will make a mess of merging between Itamar, me, master, and the next PR.